gitlab.go

```diff
@@ -187,9 +187,6 @@ func (c *Client) SetBaseURL(urlStr string) error {
 		return err
 	}
 
-	// Set the encoded opaque data
-	c.baseURL.Opaque = c.baseURL.Path
-
```

Comment:
Why are you deleting this? It's in here for a very specific reason and without it I'm pretty sure stuff will break.

Take for example the [get-single-project](http://doc.gitlab.com/ee/api/projects.html#get-single-project) call. This one (as almost all other calls) requires an ID which can take the form of `NAMESPACE/PROJECT`. This value will be places into the path, but should be treated as a single value in the URL. So this needs to be URL encodes and then put into the path.

The problem is that once you did that and then create a request for the URL it will be URL encoded again which will break the API call(s). You prevent that second encode from happening by using the opaque stuff.